src/components/Checkbox/Checkbox.module.css

```diff
@@ -18,59 +18,137 @@
   /* Magic value to center the checkbox on its label. */
   margin: 3px;
 
-  color: var(--eds-theme-color-checkbox-brand-background);
-  border: 2px solid currentColor;
+  border: calc(var(--eds-size-quarter) / 16 * 1rem) solid currentColor;
 
   /* Place the ::before content smack in the middle of the input. */
   display: grid;
   place-content: center;
+
+  border-radius: calc(var(--eds-theme-border-radius-objects-sm) * 1px);
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  
 }
 
 .checkbox__input:checked::before {
-  background-color: currentColor;
-  content: '';
-
-  /* Clip this element in the shape of a checkbox. The element's background color will be visible
-  anywhere the svg path is solid. Whatever is behind the checkbox will show through wherever the
-  svg is transparent. See https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path */
-  clip-path: path(
-    'M10.6 16.2L17.65 9.15L16.25 7.75L10.6 13.4L7.75 10.55L6.35 11.95L10.6 16.2ZM3 21V3H21V21H3Z'
-  );
+  color: var(--eds-theme-color-text-neutral-default-inverse);
+  content: '✓';
+  text-align: center;
```

Comment:
Changing up the implementation a bit here to achieve a few things:

- use the OS-specific glyph for the mark
- allow for adding color to the check itself, the background, and the border
- allow for shaping the checked state and disabled state to have rounded borders (which was a but requiring two clip-paths